compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -3489,6 +3530,34 @@ private void defineRecordTypeNode(BLangRecordTypeNode recordTypeNode, SymbolEnv
         symResolver.markParameterizedType(recordType, fieldTypes);
     }
 
+    private void resolveRestFieldType(BLangRecordTypeNode recordTypeNode, BRecordType recordType) {
```

Comment:
Do we need this method anymore?

Reply:
https://github.com/ballerina-platform/ballerina-lang/pull/35475/files#diff-fa94fe90a875c08eaf806f89f6ceb7eec7205acaf75c88a41e3f7103abfa5a52R3505 Seems it is not being used